src/status_im/data_store/accounts.cljs

```diff
@@ -9,6 +9,3 @@
 
 (defn save [account update?]
   (data-store/save account update?))
-
-(defn save-all [accounts update?]
-  (data-store/save-all accounts update?))
```

Comment:
Why is this gone with network switching?

Reply:
`src/status_im/data_store/realm/accounts.cljs` ns was touched by PR, and `save-all` from `src/status_im/data_store/realm/accounts.cljs`was used only in one place, here. So i made a small cleanup in ns that was related to PR and this affected another ns which is acceptable imo. In general i agree that touching code that is completely unrelated to PR is not desirable.